apps/epic-react/src/components/podcasts/podcast-player.tsx

```diff
@@ -20,7 +21,7 @@ const PodcastPlayer = ({episodeId}: {episodeId: string}) => {
         setState({loading: false}),
       )
     }
-  }, [theme])
+  }, [theme, episodeId])
```

Comment:
the only time we set the iframe src is in this useEffect (which is stored in a ref) so we need to include `episodeId` in the dependency array to reassign the src on navigation